nameko/rpc.py

```diff
@@ -350,10 +351,16 @@ def __call__(self, *args, **kwargs):
         reply = self._call(*args, **kwargs)
         return reply.result()
 
-    def async(self, *args, **kwargs):
+    def call_async(self, *args, **kwargs):
         reply = self._call(*args, **kwargs)
         return reply
 
+    def async(self, *args, **kwargs):  # pragma: no cover
```

Comment:
sure. the tests use [`py.test`](http://pytest.org/latest/), are you familiar with that?

`py.test --help` shows our custom options, including `--amqp-uri=AMQP_URI   The AMQP-URI to connect to rabbit with.` (admittedly not that easy to find if you don't know what you're looking for)

you can also let `tox` handle it all for you (`pip install tox; tox`). actually, i think `tox` calls a makefile, so you can probably write `make test` (or look in the `Makefile` for other targets to run subtests)

to install test dependencies you can install `nameko[dev]`, e.g. `pip install -e .[dev]` 

will try to write some of this up and add to the docs